### ISSUE TITLE
New ontology request for COVoc

### ISSUE THREAD

**Issue by zoependlington** [OBOFoundry/OBOFoundry.github.io#1338]
Use this form to register a new ontology with the OBO Foundry. Please read the instructions provided here:
http://obofoundry.org/docs/NewOntologyRegistrationInstructions.html

## Ontology title
Controlled Vocabulary for Coronavirus (COVoc)

## Requested ID space
COVOC

## Ontology location
https://github.com/EBISPOT/covoc
https://raw.githubusercontent.com/EBISPOT/covoc/master/covoc.owl

## Contact person
Name: Zoe Pendlington
Email address: zmp@ebi.ac.uk
GitHub username: zoependlington

## Issue tracker
https://github.com/EBISPOT/covoc/issues

## Ontology license

- [ ] CC0 (public domain)
- [X] CC-BY (version 3 or later)
- [ ] Other: *please specify*

## Available ontology formats
OWL, OBO, JSON

## What domain is the ontology intended to cover?
Coronavirus and COVID

## Related OBO Foundry ontologies
Chebi, CL, GO, Mondo, UBERON, CLO, HANCESTRO, HP, RO

## Intended use/related projects
COVoc is intended to provide a COVID-19 ontology to support COVID-19 literature triage.

## Data source
Hand curated spreadsheet from the Swiss Bioinformatics Institute

**Comment by ddooley** [comment]
Discussed at Corona Virus workshop.  Now part of a reconciliation effort from the workshop.  
Reservations about CIDO being there already - both developed in parallel (understandably nothing to align to).

Can a COVoc representative report here about workshop feedback, and next steps for harmonization / differentiation with CIDO?

And any objections that OBOFoundry representatives can clearly state here against adoption?

**Comment by zoependlington** [comment]
> How COVoc differs from the CIDO: Coronavirus Infectious Disease Ontology, which is already an OBO library ontology? There seems to be complete overlap, and the developing teams should at least attempt to collaborate.

The relationship of COVoc and CIDO was discussed extensively at the Workshop for Coronavirus ontologies (WCO) on the 23rd October. We will be a part of the effort that is being organised by @yongqunh to reconcile the various COVID ontologies that exist.

Reasons for the current divergence between CIDO and COVoc, in summary:

- The development of COVoc began earlier this year in March 2020, in tandem with the development of CIDO. When the CIDO paper was released in June of 2020, we had already released the beta version of COVoc, which was being used and tested in the COVID triage tool. We anticipate this duplication of COVID-related ontologies arose due to the ongoing COVID-19 crisis, which required rapid development of a solution. Now both ontologies are established, we would be very open to aligning our efforts in order to ensure future interoperability.
- One thing to note is that there is not a complete overlap between CIDO and COVoc as the primary use case for COVoc is literature triage and not ontological representation, like CIDO.

> COVoc represents Controlled Vocabulary for Coronavirus. However, OBO is for ontology, not for Controlled Vocabulary.

As discussed more extensively in the WCO workshop, the name is due to historical reasons.

In summary, COVoc was originally just a pure controlled vocabulary organised in a spreadsheet and referred to as the controlled vocabulary for COVID-19 - or COVoc. The team at the EBI then proceeded to ontologise this controlled vocabulary into an ontology, with wider OBO efforts and our own Experimental Factor Ontology.

**Comment by yongqunh** [comment]
Yes. We had an extensive discussion at the Workshop for Coronavirus ontologies (WCO) about how to reconcile and harmonize several existing COVID ontologies, inculding COVoc and CIDO. 
We have also reached out to the developers of different COVID ontologies to arrange more meetings and collaborations. 
More discussions and communications are expected to come soon.
Thanks,   
Oliver

**Comment by cmungall** [comment]
https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/docs/RegistrationChecklist.md

## scope

An ontology should have a well-defined scope

covoc mints terms like "B-cell assays", "confirmed case", "fecal sample", "nasopharyngeal sample". These are general and shouldn't be in a disease-specific ontology.

others are named deceptively generically:

```
id: COVOC:0010046
name: possible case
def: "Any person meeting the clinical criteria for COVID-19." []
```

also:

```
id: COVOC:0010014
name: isolation
def: "Used with bacteria, viruses, fungi, protozoa, and helminths for the obtaining of pure strains or for the demonstration of the presence of or identification of organisms by DNA analyses, immunologic, or other methods, including culture techniques. It is used also with biological substances and chemicals for the isolation and purification of the constituents." []
```

This is both a generic non-COVID specific term, and also confusingly labeled since in the context of pandemics "isolation" also refers to isolation of people with the disease

cell lines:

```
id: COVOC:0020006
name: F26G1
comment: hybridoma (prod. mAb against SARS-CoV)
is_a: CL:0000010
property_value: http://purl.obolibrary.org/obo/covoc/internal_covoc_id "CL_13" xsd:string
property_value: http://purl.obolibrary.org/obo/covoc/preferred_covoc_label "F26G1" xsd:string
```

This is a cell line, there is more useful info in cellosaurus
https://web.expasy.org/cellosaurus/CVCL_YZ06

I don't think cell lines should subclass CL. Why not subclass CLO? Perhaps should be added to CLO (if it is still maintained)

I see lots of chemical entities, both small molecules as well as biologics, and some mixtures (e.g. cannabis). Did you make CHEBI requests? Is there a plan to use CHEBI eventually?

I see you are modeling specific clinical trials as classes:

```
id: COVOC:0040006
name: NCT04295551 clinical trial
def: "Multicenter Clinical Study on the Efficacy and Safety of Xiyanping Injection in the Treatment of New Coronavirus Infection Pneumonia (General and Severe)" []
xref: DB:01601
is_a: NCIT:C71104
relationship: OBI:0000293 CHEBI:31781 ! has_input
relationship: OBI:0000293 CHEBI:45409 ! has_input
```

I would say this is out of scope as thing belongs in a clinical trial registry and not an ontology. However, I can see how why this is useful for you if your use case is building a dictionary for text mining. But then OBO is not really intended as a home for such dictionaries.

## naming conventions

I see lots of classes with plural names, for OBO names should be singular

## definitions

most terms are undefined

## biological problems

You have 1a, 1ab, spike - but did you know the names/uniprot xrefs you chose for those are for a bat coronavirus, not SARS-CoV-2?

This just doesn't make any sense:

```
id: COVOC:0090005
name: spike glycoprotein (BtCoV)
synonym: "S glycoprotein" BROAD []
synonym: "SARS-CoV-2 RBD" NARROW []
synonym: "spike glycoprotein" BROAD []
xref: UniProt:Q0Q475
is_a: PR:000000001
property_value: http://purl.obolibrary.org/obo/covoc/internal_covoc_id "PG_30" xsd:string
property_value: http://purl.obolibrary.org/obo/covoc/preferred_covoc_label "spike glycoprotein (BtCoV)" xsd:string
```

See https://www.uniprot.org/uniprot/Q0Q475

## general

@zoependlington :  
> One thing to note is that there is not a complete overlap between CIDO and COVoc as the primary use case for COVoc is literature triage and not ontological representation, like CIDO

Agree, the CIDO vs COVoc alignment is a bit of a distraction. Would be great if the name of ontology could better represent intention/scope :-) 


> In summary, COVoc was originally just a pure controlled vocabulary organised in a spreadsheet and referred to as the controlled vocabulary for COVID-19 - or COVoc. The team at the EBI then proceeded to ontologise this controlled vocabulary into an ontology, with wider OBO efforts and our own Experimental Factor Ontology.

it seems that it is early in the transition from CV to ontology? Would it make sense to defer registration until this transition is further along?

**Comment by paolaroncaglia** [comment]
Hi @cmungall ,

I'm aware that the COVOC case was discussed yesterday, and that you've seen the tickets we've made to address your concerns. Beyond the general issue of COVOC being or not being in scope for OBOF, I’d like to reply to a couple of specific points please:

> others are named deceptively generically:
> 
> ```
> id: COVOC:0010046
> name: possible case
> def: "Any person meeting the clinical criteria for COVID-19." []
> ```

‘possible case’, ‘probable case’ and ‘confirmed case’ are the ECDC case definitions (https://www.ecdc.europa.eu/en/covid-19/surveillance/case-definition, a dbxref for these terms), hence their relevance to mine medical literature. (I'm not challenging the question of representing these classes better, I rather mean to clarify the need for them.)

> ## biological problems
> 
> You have 1a, 1ab, spike - but did you know the names/uniprot xrefs you chose for those are for a bat coronavirus, not SARS-CoV-2?
> 
> This just doesn't make any sense:
> 
> ```
> id: COVOC:0090005
> name: spike glycoprotein (BtCoV)
> synonym: "S glycoprotein" BROAD []
> synonym: "SARS-CoV-2 RBD" NARROW []
> synonym: "spike glycoprotein" BROAD []
> xref: UniProt:Q0Q475
> is_a: PR:000000001
> property_value: http://purl.obolibrary.org/obo/covoc/internal_covoc_id "PG_30" xsd:string
> property_value: http://purl.obolibrary.org/obo/covoc/preferred_covoc_label "spike glycoprotein (BtCoV)" xsd:string
> ```
> 
> See https://www.uniprot.org/uniprot/Q0Q475

In COVoc we aimed at explicitly representing proteins from both bat and human coronaviruses as both are relevant for mining biomedical literature. So we have terms for e.g.
Spike glycoprotein (BtCoV) = COVOC:0090005
As well as
Spike glycoprotein (SARS-CoV-2) = PR:P0DTC2
If this doesn’t address your comment, please let us know. 

Thanks for your feedback,
Paola

**Comment by ddooley** [comment]
Quick note about: ‘possible case’, ‘probable case’ and ‘confirmed case’ . For ontology search engine compatibility one certainly wants such phrases - but perhaps as synonym or alternative term.  The label of the term should spell out the semantics more specifically so that it is easy in term search results to see which ones are pertinent, e.g. "possible COVID-19 case", thus reducing possibility of misuse in a sea of similar terms.

**Comment by cmungall** [comment]
Agreed!

**Comment by cmungall** [comment]
>
>
>
> You have 1a, 1ab, spike - but did you know the names/uniprot xrefs you
> chose for those are for a bat coronavirus, not SARS-CoV-2?
>
> This just doesn't make any sense:
>
> id: COVOC:0090005
>
> name: spike glycoprotein (BtCoV)
>
> synonym: "S glycoprotein" BROAD []
>
> synonym: "SARS-CoV-2 RBD" NARROW []
>
> synonym: "spike glycoprotein" BROAD []
>
> xref: UniProt:Q0Q475
>
> is_a: PR:000000001
>
> property_value: http://purl.obolibrary.org/obo/covoc/internal_covoc_id "PG_30" xsd:string
>
> property_value: http://purl.obolibrary.org/obo/covoc/preferred_covoc_label "spike glycoprotein (BtCoV)" xsd:string
>
>
> See https://www.uniprot.org/uniprot/Q0Q475
>
> In COVoc we aimed at explicitly representing proteins from both bat and
> human coronaviruses as both are relevant for mining biomedical literature.
> So we have terms for e.g.
> Spike glycoprotein (BtCoV) = COVOC:0090005
> As well as
> Spike glycoprotein (SARS-CoV-2) = PR:P0DTC2
> If this doesn’t address your comment, please let us know.
>
OK, I was looking at the entries in the COVOC namespace - it's good you
reused PRO for the SARS-CoV-2 ones

But your BtCoV entry has a synonym "SARS-CoV-2 RBD", this looked like a
mistake or confusion. maybe this is intentional, for text mining purposes?

I'd recommend making some of the assumptions a bit more explicit. I can see
that having both SARS-CoV-2 proteins and those in its most closely related
zoonotic host being useful. But why just those two?

This isn't a registry-inclusion determining issue but a general comment
that it's difficult to see how to effectively use the ontology outside the
very specific project it's been developed for as it bakes in so many
assumptions

I see your tickets addressing the issues I raised, thanks!

>

**Comment by paolaroncaglia** [comment]
Hi @cmungall ,

> OK, I was looking at the entries in the COVOC namespace - it's good you reused PRO for the SARS-CoV-2 ones But your BtCoV entry has a synonym "SARS-CoV-2 RBD", this looked like a mistake or confusion. maybe this is intentional, for text mining purposes?

That looks like a mistake where the intention might have been to add "SARS-CoV-2 RBD" (short for receptor-binding domain) as a synonym to the human entry. I'll make a ticket to look into it. Thanks for catching that.

> I'd recommend making some of the assumptions a bit more explicit. I can see that having both SARS-CoV-2 proteins and those in its most closely related zoonotic host being useful. But why just those two?

The terms that were requested were the result of a hackathon among experts, focusing on needs for COVID-19 literature triage. Our collaborators at SIB collected the requests. My assumption is that, at the time the hackathon took place, those 2 species were deemed to be the most relevant for literature searches - I guess there weren't many articles on other species at the time.    

> This isn't a registry-inclusion determining issue but a general comment that it's difficult to see how to effectively use the ontology outside the very specific project it's been developed for as it bakes in so many assumptions 

Yes, I see. The planned integration with other COVID-19 ontologies (not possible at time of COVOC creation) should address that.

Thanks,
Paola

**Comment by yongqunh** [comment]
Happy to discuss with the COVoc group. I think our CIDO ontology can help out with the integration. We may consider schedule some meeting(s) to discuss in detail. 
Thanks,
Oliver

**Comment by nlharris** [comment]
What is the status of this? Are we waiting for action from the ontology owner?

**Comment by matentzn** [comment]
I will make sure the issue is addressed when the time comes. Don't worry about it now :)

**Comment by pfabry** [comment]
Hi @zoependlington,
Since there has been no activity on this issue for quite a long time, we assume that it is no longer relevant and choose to close it to facilitate issues handling on this repository. Please let me know if there are any questions about this process and feel free to reopen it if you think it is still of interest.
